dotnet/src/Skills/Skills.Planning/SequentialPlanner/SequentialPlanner.cs

```diff
@@ -1,36 +1,46 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
+using System.IO;
 using System.Threading.Tasks;
 using Microsoft.SemanticKernel.Diagnostics;
 using Microsoft.SemanticKernel.Orchestration;
 
-namespace Microsoft.SemanticKernel.Planning.Planners;
+namespace Microsoft.SemanticKernel.Skills.Planning.SequentialPlanner;
 
 /// <summary>
 /// A planner that uses semantic function to create a sequential plan.
 /// </summary>
 public class SequentialPlanner
```

Comment:
I think we should leave this in the core and any other planners we create/experiment with go in the Microsoft.SemanticKernel.Planning package.

Reply:
@lemillermicrosoft if we are encouraging new planners to be created I would do the same approach that was done for the Services abstractions:

Example:

Having a: `SemanticKernel.Abstractions.Planning`
and `SemanticKernel.Planning.SequentialPlanner` depending on it.

Reply:
Yeah originally I had an `IPlanner` interface but was encouraged to remove it. What do you think would be appropriate to have in `SemanticKernel.Abstractions.Planning` in this case? `PlanException` maybe? Anything else?

Reply:
we can revisit the location but I would not split the planners in separate locations. The basic planner and a bunch of code that is coming need to be in the same place. Using a separate library makes it easier to support backward compatibility of the core kernel. 
If we place planners in the core, after we promote the nuget to beta or v1, it will be much harder making any change.